libFMS.F90

```diff
@@ -108,67 +107,17 @@ module fms
                                diag_integral_end
 
   !> diag manager
+  !! includes imports from submodules made public
   use diag_manager_mod, only: diag_manager_init, send_data, send_tile_averaged_data, &
                            diag_manager_end, register_diag_field, register_static_field, &
                            get_base_time, get_base_date, need_data, &
-                           diag_field_add_attribute, diag_field_add_cell_measures, &
-                           get_diag_field_id, diag_manager_set_time_end, & 
-                           diag_send_complete, diag_send_complete_instant
-  use diag_grid_mod, only: diag_grid_init, diag_grid_end, get_local_indexes, &
-                           get_local_indexes2 
-  use diag_table_mod, only: parse_diag_table
-  use diag_util_mod, only: get_subfield_size, log_diag_field_info, update_bounds, &
-                           check_out_of_bounds, check_bounds_are_exact_dynamic, &
-                           check_bounds_are_exact_static, init_file, diag_time_inc, &
-                           find_input_field, init_input_field, init_output_field, &
-                           diag_data_out, write_static, check_duplicate_output_fields, &
-                           get_date_dif, get_subfield_vert_size, sync_file_times, &
-                           prepend_attribute, attribute_init, diag_util_init
-  use diag_output_mod, only: diag_output_init, write_axis_meta_data, write_field_meta_data, &
-                           done_meta_data, get_diag_global_att, &
-                           set_diag_global_att, diag_field_write, diag_write_time
-  use diag_axis_mod, only: diag_axis_init, get_diag_axis, get_domain1d, &
```

Comment:
You need to include diag_axis_init in the public routines

Reply:
Idk how that happened, must've forgot to add the file again before the commit. Fixed now